packages/task/src/common/task-protocol.ts

```diff
@@ -27,6 +27,16 @@ export interface TaskCustomization {
     type: string;
     group?: 'build' | 'test' | 'none' | { kind: 'build' | 'test' | 'none', isDefault: true };
     problemMatcher?: string | ProblemMatcherContribution | (string | ProblemMatcherContribution)[];
+
+    /** Whether the task is a background task or not. */
+    isBackground?: boolean;
+
+    /** The other tasks the task depend on. */
+    dependsOn?: string | TaskIdentifier | Array<string | TaskIdentifier>;
+
+    /** The order the dependsOn tasks should be executed in. */
+    dependsOrder?: string;
```

Comment:
Should we use the `enum` instead (ex: `parallel` | `sequence`) ?

Reply:
fixed, done